client/my-sites/plans-features-main/hooks/data-store/use-pricing-meta-for-grid-plans.ts

```diff
@@ -53,29 +51,27 @@ const usePricingMetaForGridPlans: UsePricingMetaForGridPlans = ( {
 	withoutProRatedCredits = false,
 	storageAddOns,
 }: Props ) => {
+	// TODO: pass this in as a prop to uncouple the dependency
 	const selectedSiteId = useSelector( getSelectedSiteId ) ?? undefined;
+	// TODO: pass this in as a prop to uncouple the dependency
+	const planAvailabilityForPurchase = useCheckPlanAvailabilityForPurchase( { planSlugs } );
+
 	// pricedAPIPlans - should have a definition for all plans, being the main source of API data
 	const pricedAPIPlans = Plans.usePlans();
 	// pricedAPISitePlans - unclear if all plans are included
 	const pricedAPISitePlans = Plans.useSitePlans( { siteId: selectedSiteId } );
 	const currentPlan = Plans.useCurrentPlan( { siteId: selectedSiteId } );
 	const introOffers = Plans.useIntroOffers( { siteId: selectedSiteId } );
-	const purchasedPlan = Purchases.useSitePurchaseById( {
-		siteId: selectedSiteId,
-		purchaseId: currentPlan?.purchaseId,
-	} );
+	const purchasedPlan = Purchases.useSitePurchaseById( {} );
```

Comment:
It looks like the props of the argument, `siteId` and `purchasedId`, are accidentally removed here :) 
Without them, `purchasedPlan` will always be `undefined` since the underlying `useSitePurchases` hook won't make real request: https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/purchases/queries/use-site-purchases.ts#L39 .

Reply:
Thanks. Yes, totally unintentional change this.